### DISCUSSION TITLE
SyntaxTree plugin for ERB-formatting

### DISCUSSION THREAD

**Discussion by davidwessman** [therubyradar/submissions#146] [Submissions]
I wrote a SyntaxTree-plugin for formatting HTML-files with ERB-syntax.

After looking for a good solution I decided to try to write a plugin for SyntaxTree since I have really been enjoying that tool.
Please try it out and open issues if you encounter any.

https://github.com/davidwessman/syntax_tree-erb

**Comment by andrewmcodes** [comment]
I have talked to Kevin about SyntaxTree a few times but never thought of extending it like this. This is one of those gems I stick in a very specific folder of my bookmarks knowing one day it'll be there when I need it 🙌  Nice work @davidwessman
